onadata/libs/filters.py

```diff
@@ -357,7 +362,10 @@ def _xform_filter(self, request, view, keyword):
             xforms = xform_qs.filter(shared_data=True)
         else:
             xforms = super().filter_queryset(request, xform_qs, view) | public_forms
-        return {f"{keyword}__in": xforms}
+        return {
+            **{f"{keyword}__in": xforms},
+            **(dataview_kwargs or {})
```

Comment:
Does this work when it's not a dataview ? Don't see a default variable... won't the variable be undefined in some scenarios